Dockerfile

```diff
@@ -30,14 +30,14 @@ ARG TARGETPLATFORM TARGETOS TARGETARCH TARGETVARIANT VERSION=dev
 # to avoid https://github.com/moby/buildkit/issues/2334
 # We can use docker layer cache so the build is fast enogh anyway
 # We also use per-platform GOCACHE for the same reason.
-env GOCACHE /build/${TARGETPLATFORM}/root/.cache/go-build
```

Comment:
OMG! How did it ever work with lower-case?!

Reply:
Docker tries to interpret the first word as instruction unless it's bounded in between here doc

Reply:
Good to know! Thanks a lot for the explanation ☺️